stored_queries/taxonomy/taxonomy_search_species.yaml

```diff
@@ -0,0 +1,39 @@
+# Search for a species/strain. Similar to search_sci_name, but simpler and quicker
+name: taxonomy_search_species
+params:
+  type: object
+  required: [search_text, ts, "@taxon_coll", sciname_field]
+  properties:
+    "@taxon_coll":
+      type: string
+      title: Taxon collection name
+      examples: [ncbi_taxon, gtdb_taxon]
+    search_text:
+      type: string
+      title: Search text
+      description: Text to search on for the scientific name
+    offset:
+      type: integer
+      default: 0
+      maximum: 100000
```

Comment:
I don't think elastic will allow more than 10k will it?

Reply:
This is arango

Reply:
Ugh, brain fart